jupyter_cache/cli/commands/cmd_cache.py

```diff
@@ -66,7 +66,7 @@ def show_cache(cache_path, pk):
         click.echo("")
         return
     if paths:
-        click.echo(f"Artifacts:")
+        click.echo(f"Artifacts:")  # noqa: F541
```

Comment:
Why not just remove the `f` instead of adding the `noqa`?

Reply:
Umm. Actually I thought someone had something in mind while putting this `f` there and might come back to it. But, on second thought, we should just remove it.